dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/EventMapperTest.java

```diff
@@ -52,7 +52,7 @@
     @Mock
     private Geometry geometry;
 
-    private EventMapper mapper = Mappers.getMapper( EventMapper.class );
+    private Dxf2EventMapper mapper = Mappers.getMapper( Dxf2EventMapper.class );
```

Comment:
This test should be renamed to Dxf2EventMapperTest as its testing the old mapper. Or adapted to test the new EventMapper, unless it does not bring any value.